src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Utilities/Utilities.cs

```diff
@@ -153,7 +155,7 @@ public static string GetDefaultPath()
                 throw new Exception("Invalid ARM ID");
             }
 
-            for (var i = 1; i < armFields.Length; i = i + 2)
+            for (var i = 0; i < armFields.Length; i = i + 2)
```

Comment:
@viverm shouldn't this _always_ fail with an `IndexOutOfRangeException`?

For example, if `armFields` has five elements, this loop will get the following elements:

`i == 0`: 1st (idx 0) and 2nd (idx 1)
`i == 2`: 3rd (idx 2) and 4th (idx 3)
`i == 4`: 5th (idx 4) and 6th (idx 5 **DOESN'T EXIST!**)

Reply:
here we were expecting things in pair  like 1st will be the 2nd the value .
We are using the other utility method to get the  value .this call is not used anymore .
Marked this as part of cleanup activity,